### PR TITLE
MICROS-6602 internaldns webhook scope fix

### PR DESCRIPTION
**Pull request by marciogh**

### CODE REVIEW

File: pkg/execution/svccatadmission/internaldns.go [hunk 1]
```diff
@@ -18,6 +18,10 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+func isInternalDNSServiceClass(serviceInstance sc_v1b1.ServiceInstance) bool {
```
Comment (ash2k) [inline pkg/execution/svccatadmission/internaldns.go]: Pass a pointer here. By conventions API type objects are always passed by pointer. And it makes sense - they may be big.

Comment (marciogh) [reply pkg/execution/svccatadmission/internaldns.go]: done!

File: pkg/execution/svccatadmission/internaldns_test.go [hunk 2]
```diff
@@ -176,6 +176,22 @@ func TestInternalDNSAdmitFunc(t *testing.T) {
...
+		{
+			"not internaldns create",
+			buildAdmissionReview("", k8s.ServiceInstanceGVR, admissionv1beta1.Create, buildServiceInstance(
+				t, "otherClassExternalID", "otherPlanExternalID", apiinternaldns.Spec{}),
+			),
+			buildAdmissionResponse(true, 0, metav1.StatusReasonUnknown, nil, "requested ServiceInstance is not InternalDNS type"),
+			false,
+		},
+		{
+			"not internaldns create",
```
Comment (nilebox) [inline pkg/execution/svccatadmission/internaldns_test.go]: update

Comment (marciogh) [reply pkg/execution/svccatadmission/internaldns_test.go]: fixed (nice catch)

**Pull request merged**